src/warp.jl

```diff
@@ -65,7 +49,7 @@ julia> indices(img)
 julia> tfm = recenter(RotMatrix(pi/4), center(img))
 AffineMap([0.707107 -0.707107; 0.707107 0.707107], [347.01,-68.7554])
 
-julia> imgw = warp(img, tfm);
+julia> imgw = warp(img, inv(tfm));
```

Comment:
I think it would be better to pass `tfm` directly. You could change the angle to `-pi/4` above.